src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -348,6 +348,28 @@ WARNING: It is recommended to use an enabled-GPU node to run the deep learning t
 The `machine-learning` bucket contains diverse generic machine learning tasks that enable you to easily compose workflows for predictive models learning and testing. This bucket can be easily customized according to your needs.
 This bucket offers different options, you can customize it by adding new tasks or update the existing tasks.
 
+ALl the tasks included in this bucket have som common task variables that we describe in the table below.
+
+*Common Task Variables:*
+
+.Common_Machine_Learning_Tasks variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `Docker_ENABLED`
+|If True,  the tasks will be executed on a Docker container. link:https://hub.docker.com/r/activeeon/dlm3/[activeeon/dlm3] image is used by default. If False, the required libraries should be installed in the different nodes that will be used.
+| Boolean (default=True)
+| `Docker_IMAGE`
+| Specifies the docker image that will be used to execute the task.
+| String (default="activeeon/dlm3"
+| `TASK_ENABLED`
+| If False, the will be ignored, it will not be executed.
+| Boolean (default=True)
+| `LIMIT_OUTPUT_VIEW`
+| Specifies how many rows of the CSV will be previewed in the browser to check each task results.
```

Comment:
| Specifies how many rows of the CSV will be previewed in the browser to check each task results. 
Specifies how many rows of the data frame will be previewed in the browser to check each task results.